src/style-guide.scrbl

```diff
@@ -277,6 +316,84 @@ cases (Animal) a:
 end
 }
 
+@section{Boolean Expressions}
+
+@subsection{Ask Blocks}
+
+If you have an @tt{if} block containing a lot of conditionals (greater than 3
+branches), you should convert it to an @tt{ask} block. This aligns your
+conditionals along the left side of the block, which makes your code look
+cleaner and more readable.
+
+For example, this @tt{if} block is very verbose:
+
+@codedisp{
+fun is-this-noah(str :: String) -> String:
+  if str == "Noah":
+    "Yes"
+  else if str == "Nosh":
+    "Close"
+  else if str == "No":
+    "ah"
+  else if str == "Noa":
+    "I don’t think so"
+  else:
+    "No"
+  end
+end
+}
+
+However, we can make it more concise using an @tt{ask} block:
+
+@codedisp{
+fun is-this-noah(str :: String) -> String:
+  ask:
+    | str == "Noah" then: "Yes"
+    | str == "Nosh" then: "Close"
+    | str == "No" then: "ah"
+    | str == "Noa" then: "I don’t think so"
+    | otherwise: "No"
+  end
+end
+}
+
+@subsection{Concise Conditionals}
+
+You should never determine the boolean value of something by doing:
+
+@codedisp{
+if some-boolean == true:
+  ...
+else:
+  ...
+end
+}
+
+Instead, you should use:
+
+@codedisp{
+if some-boolean:
+  ...
+else:
+  ...
+end
+}
+
+Similarly, do not use an @tt{if} expression to evalulate a predicate, and
+then return @tt{true} or @tt{false}. For example, this would be
+considered incorrect:
```

Comment:
As this is about style, I feel it's weird to say that something is _incorrect_. Perhaps use "inappropriate" (or another word) instead?